src/components/HoverButton.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import Fab from "@material-ui/core/Fab";
+import BugReportIcon from "@material-ui/icons/BugReport";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import CodeIcon from "@material-ui/icons/Code";
+import Link from "@material-ui/core/Link";
+import { makeStyles } from "@material-ui/core/styles";
+
+const usestyles = makeStyles({
+  container: {
+    display: "flex",
+    flexDirection: "column",
+    margin: 0,
+    top: "auto",
+    right: 20,
+    bottom: 20,
+    left: "auto",
+    position: "fixed",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+
+  icons: {
+    marginBottom: "10px",
+  },
+});
```

Comment:
I think this should be on a separate file perhaps you could even use `utils/styles`

Reply:
I should export these styles from utils/styles and then use them?

Reply:
That file carry styles, it would be less clogged; if you do it this way make sure to change the variable name to make reference to the button.

Reply:
Ok, Should I make another pull request?

Reply:
No, just modify the files and push to your repo again

Reply:
Done